src/containers/App.tsx

```diff
@@ -66,17 +66,14 @@ function App() {
           />
           <Route
             exact
-            path="/patients/edit/:ccpId/:patientId"
-            component={(props) => <PatientProfilePage mode="edit" {...props} />}
+            path="/events/:eventId/ccps/:ccpId/patients/new/:barcodeValue?"
+            component={(props) => <PatientProfilePage mode="new" {...props} />}
           />
           <Route
             exact
-            path="/patients/new/:ccpId/:barcodeValue?"
-            component={(props) => <PatientProfilePage mode="new" {...props} />}
+            path="/events/:eventId/ccps/:ccpId/patients/:patientId"
+            component={(props) => <PatientProfilePage mode="edit" {...props} />}
           />
-          <Route path="/">
-            <Redirect to="/events" />
-          </Route>
```

Comment:
let's leave this in for now - this will be changed once we figure out authentication